### DISCUSSION TITLE
Any Sugggestions for Books I can Read on ESP32 Development ?

### DISCUSSION THREAD

**Discussion by daveclifford** [DCC-EX/EX-T3-WiFi#46] [General]
Do you have any suggestions for documentation on ESp32 development and upload?

**Comment by alex-code** [comment]
I'm self taught by example and docs so don't have any references I can mention.
The Arduino wrapper for IDF is pretty good so any Arduino examples you find can usually be ported.

**Reply by daveclifford** [reply]
Alex,

I just found the resource **http://esp32.net/** - Very profound for all they have cataloged.

Dave
